catboost/python-package/ut/medium/test.py

```diff
@@ -3664,3 +3665,40 @@ def test_eval_set_with_no_target_with_eval_metric(task_type):
     )
     with pytest.raises(CatboostError):
         model.fit(train_pool, eval_set=eval_set_pool)
+
+
+def test_model_comparison():
+    def fit_model(value):
+        train_df = titanic()[0].fillna(value)
```

Comment:
It's better to use a specific small dataset here that is commited right into repository and used in other tests. Like https://github.com/catboost/catboost/tree/master/catboost/pytest/data/adult. See here for example - https://github.com/catboost/catboost/blob/1aa0ba06cceef82cd4dc64cad084b293ae143839/catboost/python-package/ut/medium/test.py#L196